test/e2e/page-objects/pages/home/asset-list.ts

```diff
@@ -506,6 +511,18 @@ class AssetListPage {
     );
   }
 
+  async checkPriceChartLoaded(assetAddress: string): Promise<void> {
+    console.log(`Verify the price chart is loaded`);
+    await this.driver.waitUntil(
+      async () => {
+        return await this.driver.isElementPresentAndVisible(
+          `[data-testid="token-increase-decrease-percentage-${assetAddress}"]`,
+        );
+      },
+      { timeout: 10000, interval: 100 },
+    );
```

Comment:
1. Could we move the selector on top, with the rest of selectors

```
  private readonly tokenPercentage = (assetAddress: string) =>
    `[data-testid="token-increase-decrease-percentage-${assetAddress}"]`;
```

2. Instead of waitUntil, could we just do this for simplicity?

```suggestion
  await driver.waitForSelector(this.tokenPercentage(assetAddress));
```